frontend-react/src/resources/AuthResource.ts

```diff
@@ -6,7 +6,7 @@ import {
 } from "../contexts/SessionStorageTools";
 
 export default class AuthResource extends Resource {
-    pk(parent?: any, key?: string): string | undefined {
+    pk(_parent?: any, _key?: string): string | undefined {
```

Comment:
FYI, prefixing an `_` tells the eslinter you KNOW it's unused and it's ok.

Reply:
Huh! Interesting. Is this a standard thing among engineers or is this just the linter providing an override?